native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.android.js

```diff
@@ -121,7 +122,7 @@ const ALERT_TYPE_STYLES = {
 const DismissableButton = ({ dismissButtonLabel, onDismiss }) => (
   <View style={styles.dismissableContainer}>
     <View style={styles.dismissButtonContainerRadius}>
-      <TouchableNativeFeedback
+      <BpkTouchableNativeFeedback
         accessibilityComponentType="button"
         accessibilityLabel={dismissButtonLabel}
         background={TouchableNativeFeedback.SelectableBackgroundBorderless()}
```

Comment:
This line (along with the import of `TouchableNativeFeedback` in this file) should be removed.

Reply:
ach that explains that weird test